internal/mode/static/nginx/config/servers.go

```diff
@@ -38,58 +39,82 @@ var baseHeaders = []http.Header{
 	},
 }
 
-func executeServers(conf dataplane.Configuration) []byte {
-	servers := createServers(conf.HTTPServers, conf.SSLServers)
+func executeServers(conf dataplane.Configuration) ([]http.Server, map[string][]http.RouteMatch) {
+	servers, httpMatchPairs := createServers(conf.HTTPServers, conf.SSLServers)
 
-	return execute(serversTemplate, servers)
+	return servers, httpMatchPairs
 }
 
-func createServers(httpServers, sslServers []dataplane.VirtualServer) []http.Server {
+func createServers(httpServers, sslServers []dataplane.VirtualServer) (
+	[]http.Server,
+	map[string][]http.RouteMatch,
+) {
```

Comment:
The pattern to break up longs lines is to put the function arguments on newlines, not the return values. (same comment for subsequent instances like this). Some of your function signatures also aren't very long and probably don't need to be broken up.